lib/streamlit/elements/media.py

```diff
@@ -52,13 +55,22 @@ def audio(
         data : str, bytes, BytesIO, numpy.ndarray, or file opened with
                 io.open().
             Raw audio data, filename, or a URL pointing to the file to load.
-            Numpy arrays and raw data formats must include all necessary file
+            Raw data formats must include all necessary file
             headers to match specified file format.
+            In case of numpy array:
+            * Numpy 1d array containing the desired waveform (mono)
+            * Numpy 2d array containing waveforms for each channel.
+              Shape=(NCHAN, NSAMPLES). For the standard channel order, see
+              http://msdn.microsoft.com/en-us/library/windows/hardware/dn653308(v=vs.85).aspx
         format : str
             The mime type for the audio file. Defaults to 'audio/wav'.
             See https://tools.ietf.org/html/rfc4281 for more info.
         start_time: int
             The time from which this element should start playing.
+        sample_rate: int or None
+            The sample rate of the raw audio data, number of samples per second.
```

Comment:
The sample rate of the audio data in samples per second. Only required if `data` is a numpy array.